flutter-studio/src/io/flutter/project/FlutterProjectCreator.java

```diff
@@ -241,15 +241,18 @@ public void createProject() {
       Messages.showErrorDialog(projectToClose, message, ActionsBundle.message("action.NewDirectoryProject.title"));
       return;
     }
+    final File baseFile = new File(location, myModel.projectName().get());
+    //noinspection ResultOfMethodCallIgnored
+    baseFile.mkdirs();
     final VirtualFile baseDir = ApplicationManager
-      .getApplication().runWriteAction((Computable<VirtualFile>)() -> LocalFileSystem.getInstance().refreshAndFindFileByIoFile(location));
+      .getApplication().runWriteAction((Computable<VirtualFile>)() -> LocalFileSystem.getInstance().refreshAndFindFileByIoFile(baseFile));
     if (baseDir == null) {
       LOG.error("Couldn't find '" + location + "' in VFS");
```

Comment:
I believe we have LOG.error() set up to show a dialog to the user. We probably want to revisit that; in the meantime, I'd recommend LOG.warn().

Reply:
(just noticed that this is not part of your PR; I'll make a note to look into why we're showing a dialog on log.error())